Framework/DataObjects/test/MDEventWorkspaceTest.h

```diff
@@ -295,6 +295,39 @@ class MDEventWorkspaceTest : public CxxTest::TestSuite {
   }
 
   //-------------------------------------------------------------------------------------
+  /** Get the signal at a given coord or 0 if masked */
+  void test_getSignalWithMaskAtCoord() {
+    MDEventWorkspace3Lean::sptr ew =
+        MDEventsTestHelper::makeMDEW<3>(4, 0.0, 4.0, 1);
+    coord_t coords1[3] = {0.5, 0.5, 0.5};
+    coord_t coords2[3] = {2.5, 2.5, 2.5};
+    ew->addEvent(MDLeanEvent<3>(2.0, 2.0, coords2));
+
+    std::vector<coord_t> min;
+    std::vector<coord_t> max;
+
+    min.push_back(0);
+    min.push_back(0);
+    min.push_back(0);
+    max.push_back(1.5);
+    max.push_back(1.5);
+    max.push_back(1.5);
```

Comment:
You could use initializer list now, but that is not really important here. See [here](http://en.cppreference.com/w/cpp/language/list_initialization)